ProcessesApi.Tests/V1/E2ETests/Steps/UpdateSoleToJointProcessSteps.cs

```diff
@@ -95,7 +95,7 @@ public async Task ThenTheIncomingTenantIdIsAddedToRelatedEntities(UpdateProcessQ
             var dbRecord = await _dbFixture.DynamoDbContext.LoadAsync<ProcessesDb>(request.Id).ConfigureAwait(false);
 
             var incomingTenantId = Guid.Parse(requestBody.FormData[SoleToJointFormDataKeys.IncomingTenantId].ToString());
-            dbRecord.RelatedEntities.Should().Contain(incomingTenantId);
+            dbRecord.RelatedEntities.Should().Contain(x => x.Id == incomingTenantId);
```

Comment:
Are you going to assert on the rest of the values (e.g. subType = householdMember or incomingTenant or something)?